File: clean.py

import re
with open('src/app/(admin)/settings/leaderboard/item-results.tsx', 'r', encoding='utf-8') as f:
    c = f.read()

c = re.sub(r'\{itemCategoryCodes\.get\(group\.item_id\)\?\.map\(\(code: string\) => \{.*?\}\)\}', '', c, flags=re.DOTALL)

with open('src/app/(admin)/settings/leaderboard/item-results.tsx', 'w', encoding='utf-8') as f:
    f.write(c)

File: clean.js
const fs = require('fs'); let c = fs.readFileSync('src/app/(admin)/settings/leaderboard/item-results.tsx', 'utf8');
c = c.replace(/\\{itemCategoryCodes\\.get\\(group\\.item_id\\)\\?\\.map\\(\\(code: string\\) => \\{[\\s\\S]*?\\}\\)}\\)/g, '');
fs.writeFileSync('src/app/(admin)/settings/leaderboard/item-results.tsx', c);

